atm.js

```diff
@@ -1,21 +1,59 @@
-var checkingBalance = document.querySelector("div#checking_balance");
-checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
-
-var checkingDeposit = function(amount){
-  checkingBalance = parseInt( checkingBalance.innerHTML.replace("$", "") );
-
+var checkingBalance = $("#checking_balance").html();
+
+function checkingDeposit(){
+  if (!isNaN(parseInt($("#checkingInput").val()))){
+    var newDeposit = parseInt($("#checkingInput").val());
+    var balance = parseInt($("#checking_balance").html().replace("$",""));
+    var newBalance = newDeposit + balance;
+    $("#checking_balance").text("$" + newBalance);
+    $("#checking_balance").removeClass("zero");
+    $("#checkingInput").val("");
+ }
 }
 
-function checkingWithdrawal(amount){
 
+function checkingWithdrawal(){
+  var newWithdrawal = parseInt($("#checkingInput").val());
+  var balance = parseInt($("#checking_balance").html().replace("$",""));
+  var savingsBalance = parseInt($("#savings_balance").html().replace("$",""));
+  var totalBalance = balance + savingsBalance;
+  if( newWithdrawal < balance ){
+    var newBalance = balance - newWithdrawal;
+    $("#checking_balance").text("$" + newBalance);
+  } else if ( newWithdrawal <= totalBalance){
+    $("#checking_balance").text("$0");
+    $("#checking_balance").addClass("zero");
+    var newBalance = totalBalance - newWithdrawal;
+    $("#savings_balance").text("$" + newBalance);
+  }
```

Comment:
Probably want to handle the `else` case here by notifying that the withdrawal couldn't be made due to not enough in savings + checking!